gadget/update.go

```diff
@@ -665,6 +667,143 @@ func DiskTraitsFromDeviceAndValidate(expLayout *LaidOutVolume, dev string, opts
 	}, nil
 }
 
+// unable to proceed with the gadget asset update, but not fatal to the refresh
+// operation itself
+var errSkipUpdateProceedRefresh = errors.New("could not identify disk for gadget asset update")
+
+// buildNewVolumeToDeviceMapping builds a DiskVolumeDeviceTraits for only the
+// system-boot role containing volume when we cannot load an existing traits
```

Comment:
found it a bit convoluted, how about:
```suggestion
// buildNewVolumeToDeviceMapping builds a DiskVolumeDeviceTraits only for the
// volume containing the system-boot role, when we cannot load an existing traits
```